### PR TITLE
Fix compile errors on different runtime

### PR DESCRIPTION
**Pull request by billy1624**

### CODE REVIEW

File: sea-orm-cli/Cargo.toml [hunk 1]
```diff
@@ -29,7 +29,9 @@ sea-schema = { version = "^0.2.4", default-features = false, features = [
 sqlx = { version = "^0.5", default-features = false, features = [ "mysql", "postgres" ] }
 
 [features]
-default = [ "runtime-async-std-native-tls" ]
+# Warning: removing this line means one cannot simply run `cargo install sea-orm-cli`.
+# They have to run `cargo install sea-orm-cli --features runtime-async-std-native-tls`
+# default = [ "runtime-async-std-native-tls" ]
```
Comment (tyt2y3) [inline sea-orm-cli/Cargo.toml:34]: sea-orm-cli is an app. People shouldn't care about which runtime we are using.
I think we should ship a default runtime.
If it's so problematic to be in the same workspace, we can split it out to its own workspace, or even another repo.

Comment (billy1624) [reply sea-orm-cli/Cargo.toml:34]: Agree, split it out to its workspace worked!

Comment (tyt2y3) [reply sea-orm-cli/Cargo.toml:34]: Cool. Let's try.

**Comment by billy1624** [comment]
Check this @tyt2y3

**Comment by tyt2y3** [comment]
Nice

**Pull request merged**